src/media/js/main.js

```diff
@@ -215,6 +180,10 @@ require([
             }
         }
 
+        require(['update_banner'], function(update_banner) {
```

Comment:
I think it's fine to invoke this straight from update_banner.js, and then make it a dependency of main. Making a module a dependency of main to initialize global handlers and initialize startup stuff is common.